model/group_impl.go

```diff
@@ -33,6 +36,39 @@ func (repo *EntRepository) CreateGroup(ctx context.Context, name string, descrip
 	return ConvertEntGroupToModelGroup(created), nil
 }
 
+func (repo *EntRepository) GetMembers(ctx context.Context, groupID uuid.UUID) ([]*User, error) {
+	members, err := repo.client.Group.Query().Where(group.IDEQ(groupID)).QueryUser().All(ctx)
+	if err != nil {
+		return nil, err
+	}
+	modelmembers := []*User{}
+	for _, member := range members {
+		modelmembers = append(modelmembers, ConvertEntUserToModelUser(member))
+	}
+	return modelmembers, nil
+}
+
+func (repo *EntRepository) PostMember(ctx context.Context, groupID uuid.UUID, userID uuid.UUID) (*User, error) {
```

Comment:
model層では`Post`ではなく`Create`としてください。
```suggestion
func (repo *EntRepository) CreateMember(ctx context.Context, groupID uuid.UUID, userID uuid.UUID) (*User, error) {
```